src/Features/ParentStatus.php

```diff
@@ -164,15 +164,44 @@ public function updateParentStatus(CommonITILObject $parentitem, array $input):
             }
         }
 
-        if (
-            !$is_set_pending
-            && !empty($this->fields['begin'])
-            && $parentitem->isStatusExists(CommonITILObject::PLANNED)
-            && (($parentitem->fields["status"] == CommonITILObject::INCOMING)
-              || ($parentitem->fields["status"] == CommonITILObject::ASSIGNED)
-              || $needupdateparent)
-        ) {
-            $input['_status'] = CommonITILObject::PLANNED;
+        if (!$is_set_pending) {
+            if (
+                !empty($this->fields['begin'])
+                && $parentitem->isStatusExists(CommonITILObject::PLANNED)
+                && (($parentitem->fields["status"] == CommonITILObject::INCOMING)
+                    || ($parentitem->fields["status"] == CommonITILObject::ASSIGNED)
+                    || $needupdateparent)
+            ) {
+                $input['_status'] = CommonITILObject::PLANNED;
+            } elseif ($parentitem->fields["status"] == CommonITILObject::PLANNED) {
```

Comment:
This method may also be executed, for instance, from the `ITILFollowup` context. In this case, the `begin` field would not exist and a SQL error will probably be raised.

A test case should probably be added to ensure that adding a followup on a ticket that has the `CommonITILObject::PLANNED`  status will not fail.

Also, should the parent status be updated if the `begin` filed was not changed ?

```suggestion
            } elseif (
                $parentitem->fields["status"] == CommonITILObject::PLANNED
                && in_array('begin', $this->updates)
                && $this->isField('begin')
            ) {
```

Reply:
Applied in [2b2bd92](https://github.com/glpi-project/glpi/pull/16895/commits/2b2bd92bf33fc7912ddebe48f61fb0fdfca24017) :+1: